data/json/items/ranged.json

```diff
@@ -5113,5 +5113,33 @@
     "burst": 0,
     "clip_size": 0,
     "reload": 0
-  }
+  },
+  {
+  "id": "Abzats",
+  "type": "GUN",
+  "symbol": "(",
+  "color": "dark_gray",
+  "name": "Heavy Automatic Shotgun",
+  "description": "Modified from the massive M2 Browning, this heavy machine gun has been re-chambered and re-bored for shotgun shells, and completely redesigned to be wielded unmounted by one user.",
+  "price": 750000,
+  "material": ["hardsteel", "wood"],
+  "flags": ["MODE_BURST", "RELOAD_EJECT"], 
+  "skill": "shotgun",
+  "ammo": "shot",
+  "weight": 16500,
+  "volume": 18,
+  "bashing": 12,
+  "cutting": 0,
+  "to_hit": -1,
+  "dispersion": 150,
+  "sight_dispersion": 75,
+  "aim_speed": 5,
+  "recoil": 180,
+  "durability": 8,
```

Comment:
Given the fact that it's using the M2 Browning, which is an incredibly solid and reliable heavy machine gun as a base and been converted to fire much lighter shotgun shells instead of it's usual payload, it makes sense that it would maintain a certain level of reliability. But then you also have to take into account the fact a survivor hand machined and custom built the rest of it's assembly out of scrape and spare parts. I suppose 6, maybe 5 sounds reasonable then?. I don't really know how durability and it's value corresponds to an item's actual resilience to damage or extended use without maintenance.